torch/nn/utils/clip_grad.py

```diff
@@ -8,15 +9,17 @@ def clip_grad_norm_(parameters, max_norm, norm_type=2):
     concatenated into a single vector. Gradients are modified in-place.
 
     Arguments:
-        parameters (Iterable[Tensor]): an iterable of Tensors that will have
-            gradients normalized
+        parameters (Iterable[Tensor] or Tensor): an iterable of Tensors or a
+            single Tensor that will have gradients normalized
         max_norm (float or int): max norm of the gradients
         norm_type (float or int): type of the used p-norm. Can be ``'inf'`` for
             infinity norm.
 
     Returns:
         Total norm of the parameters (viewed as a single vector).
     """
+    if torch.is_tensor(parameters):
```

Comment:
Should use `isinstance(parameters, torch.Tensor)` instead.

Reply:
I'll submit a new PR.